cli/src/main.rs

```diff
@@ -231,8 +278,8 @@ fn main() -> Result<()> {
                         .required(true)
                         .help("Amount to send"),
                 ])
-                .help("Transfer Solana token claimable by Ethereum users"),
-            SubCommand::with_name("claim").args(&[
+                .help("Sents some amount of tokens of specified mint to the Solana account associated with Etherium address."),
```

Comment:
`Sends`, I suggest checking all the strings in Google Docs or some other editor checking orthography and syntax.